src/docker/composebuilder/service_test.go

```diff
@@ -1,292 +0,0 @@
-				Expect(err).NotTo(HaveOccurred())
-				serviceConfigs, err := config.GetServiceConfigs(appDir, appConfig)
-				Expect(err).NotTo(HaveOccurred())
-				serviceData := appConfig.Services
-				serviceRole := "users-service"
-				buildMode := composebuilder.BuildMode{
-					Type:        composebuilder.BuildModeTypeLocal,
-					Environment: composebuilder.BuildModeEnvironmentDevelopment,
-				}
-				serviceEndpoints = map[string]*composebuilder.ServiceEndpoints{
-					"html-server":      &composebuilder.ServiceEndpoints{},
-					"api-service":      &composebuilder.ServiceEndpoints{},
-					"external-service": &composebuilder.ServiceEndpoints{},
-					"users-service":    &composebuilder.ServiceEndpoints{},
-					"todo-service":     &composebuilder.ServiceEndpoints{},
-				}
-				dockerConfigs, err = composebuilder.GetServiceDockerConfigs(appConfig, serviceConfigs[serviceRole], serviceData[serviceRole], serviceRole, appDir, buildMode, serviceEndpoints)
-				Expect(err).NotTo(HaveOccurred())
-			})
-
-			var _ = AfterEach(func() {
-				err := os.Unsetenv("EXOSPHERE_SECRET")
-				Expect(err).NotTo(HaveOccurred())
-			})
-
-			It("compiles development variables", func() {
-				expectedVars := map[string]string{
-					"ENV1":             "value1",
-					"ENV2":             "value2",
-					"ENV3":             "dev_value3",
-					"EXOSPHERE_SECRET": "exosphere-value",
-				}
-				actualVars := dockerConfigs["users-service"].Environment
-				for k, v := range expectedVars {
-					Expect(actualVars).Should(HaveKeyWithValue(k, v))
-				}
-			})
-		})
-
-		var _ = Describe("shared docker configs", func() {
-			var dockerConfigs types.DockerConfigs
-			var serviceEndpoints map[string]*composebuilder.ServiceEndpoints
-
-			var _ = BeforeEach(func() {
-				appDir := helpers.GetTestApplicationDir("complex-setup-app")
-				appConfig, err := types.NewAppConfig(appDir)
-				Expect(err).NotTo(HaveOccurred())
-				serviceConfigs, err := config.GetServiceConfigs(appDir, appConfig)
-				Expect(err).NotTo(HaveOccurred())
-				serviceData := appConfig.Services
-				serviceRole := "users-service"
-				buildMode := composebuilder.BuildMode{
-					Type:        composebuilder.BuildModeTypeLocal,
-					Environment: composebuilder.BuildModeEnvironmentDevelopment,
-				}
-				serviceEndpoints = map[string]*composebuilder.ServiceEndpoints{
-					"html-server":      &composebuilder.ServiceEndpoints{},
-					"api-service":      &composebuilder.ServiceEndpoints{},
-					"external-service": &composebuilder.ServiceEndpoints{},
-					"users-service":    &composebuilder.ServiceEndpoints{},
-					"todo-service":     &composebuilder.ServiceEndpoints{},
-				}
-				dockerConfigs, err = composebuilder.GetServiceDockerConfigs(appConfig, serviceConfigs[serviceRole], serviceData[serviceRole], serviceRole, appDir, buildMode, serviceEndpoints)
-				Expect(err).NotTo(HaveOccurred())
-			})
-
-			It("should not include the docker config that is defined in application.yml", func() {
```

Comment:
Got rid of this test, don't think it's relevant anymore after we simplified dependencies